ush/setup.py

```diff
@@ -1270,250 +1306,80 @@ def setup():
     #
     # ----------------------------------------------------------------------
     #
-    global RUN_TASK_MAKE_GRID, RUN_TASK_MAKE_OROG, RUN_TASK_MAKE_SFC_CLIMO
     global GRID_DIR, OROG_DIR, SFC_CLIMO_DIR
-    global RUN_TASK_VX_GRIDSTAT, RUN_TASK_VX_POINTSTAT, RUN_TASK_VX_ENSGRID
 
     #
-    # -----------------------------------------------------------------------
-    #
-    # Make sure that DO_ENSEMBLE is set to TRUE when running ensemble vx.
+    # If RUN_TASK_MAKE_GRID is set to False, the workflow will look for
```

Comment:
@christinaholtNOAA I would agree that the config.nco.yaml should have a workflow_switches section to make it easy to exclude or include tasks in the setup.

Reply:
The NCO configuration is no more associated with pregenerated grids though. So if I tell it to set `RUN_TASK_MAKE_GRID=FALSE` I am sure it will cause problems when trying to run a `community` test case in NCO mode. However, I could do something like that specifically for `machine=WCOSS2` assuming anything run on wcoss2 is in NCO mode -- actually did something similar in the rocoto xml before but got rid of it later.
@MatthewPyle-NOAA Indeed the default values of those are TRUE so definately need to be set to false. I guess i've never run that test case before.

Reply:
@MatthewPyle-NOAA Good catch! Not only is the `config.nco.yaml` that needs those but also 4 or so other tests cases in WE2E that run in NCO mode needed it. They must have run in commmunity mode in the regression test...